tfhub_dev/assets/intel/models/rfcn_resnet101/fp32/1.md

```diff
@@ -0,0 +1,29 @@
+# Module intel/rfcn_resnet101/fp32/1
+Object detection model
+
+<!-- asset-path: https://storage.googleapis.com/public-artifacts/intelai_public_models/tfhub/rfcn_resnet101_fp32_coco_pretrained_model_tfhub.tar.gz -->
+<!-- module-type: image-object-detection -->
+<!-- dataset: Coco -->
+<!-- language: en -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
```

Comment:
Wondering here since the example uses hub.Module.

If the model is really a TF1 style hub.Module, this line should be:
<!-- format: hub -->